src/Models/Booking.php

```diff
@@ -187,7 +187,7 @@ public function status()
 
     public function bookingCategory()
     {
-        return $this->belongsTo(BookingCategory::class);
+        return $this->belongsTo(app('booking_category'));
```

Comment:
Can you check that this one is also included in tipoff/support config file along with the other classes in this pacakge?

Reply:
Nevermind, it is there.